install/09_alertrules.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: cluster-autoscaler-rules
+  namespace: openshift-machine-api
+spec:
+  groups:
+    - name: general.rules
+      rules:
+        - alert: ClusterAutoscalerOperatorDown
+          annotations:
+            message: ClusterAutoscalerOperator
+              discovery.
+          expr: |
+            absent(up{job="cluster-autoscaler-operator"} == 1)
+          for: 5m
```

Comment:
why is this `for` value different everywhere
https://github.com/openshift/cluster-machine-approver/blob/master/manifests/0000_90_cluster-machine-approver_04_alertrules.yaml#L19

https://github.com/openshift/machine-api-operator/blob/master/install/0000_90_machine-api-operator_04_alertrules.yaml#L46